examples/inference-deployments/mpt/README.md

```diff
@@ -1,22 +1,57 @@
-## Model Description
+## Inference with MPT-7B 
 
-A state-of-the-art 6.7B parameter instruction finetuned language model trained by MosaicML. Check out the [MPT-Instruct huggingface repo](https://huggingface.co/spaces/mosaicml/mpt-7b-instruct) for more information.
+[MosaicML’s inference service](https://www.mosaicml.com/blog/inference-launch) allows users to deploy their ML models and run inference on them. In this folder, we provide an example of how to use MPT-7B, a state-of-the-art 6.7B parameter instruction fine-tuned language model trained by MosaicML for inference. Check out the [MPT-Instruct huggingface repo](https://huggingface.co/spaces/mosaicml/mpt-7b-instruct) for more information about the model.
```

Comment:
this text is just about instruct, but the example is about all the variants. so lets change the text to mention all the variants/not only talk about the instruct model